radar/tools/yarn-classic.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Yarn Classic"
+ring: hold
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Being on hold, it's useful to say why. Even a silly "we just use Yarn > 3"